src/local/butler/package.py

```diff
@@ -89,7 +89,7 @@ def package(revision,
   appengine.symlink_dirs()
 
   _, ls_files_output = common.execute('git -C . ls-files', print_output=False)
-  file_paths = ls_files_output.splitlines()
+  file_paths = [path.decode('utf-8') for path in ls_files_output.splitlines()]
```

Comment:
Question: Should you decode the output first, then splitlines ?

Reply:
No real difference here, so leaving as is.